neard/src/config.rs

```diff
@@ -694,6 +694,13 @@ fn state_records_account_with_key(
     ]
 }
 
+/// Generate validator key if account is not `None`.
+fn generate_validator_key(account_id: &str, path: &Path) {
+    let signer = InMemoryValidatorSigner::from_random(account_id.to_string(), KeyType::ED25519);
+    info!(target: "near", "Use key {} for {} to stake.", signer.public_key(), account_id);
+    signer.write_to_file(path);
```

Comment:
I suggest we either unwrap/expect, log a warning on error, or return the error since it is not uncommon to see various permission errors or out-of-disk states and it would be quite surprising to keep things going without the key.

Reply:
This function doesn't return any error. The function would panic if writing to file failed. See `KeyFile::write_to_file`

Reply:
I see. LGTM